docs/usage.rst

```diff
@@ -367,6 +367,13 @@ deck-chore's behaviour is defined by these environment variables:
 
     The URL of the Docker daemon to connect to.
 
+.. envvar:: STDERR_LEVEL
+
+    default: ``NOTSET``
+
+    Redirect any deck-chores-internal log above level STDERR_LEVEL to STDERR, string or integer required.
```

Comment:
1) it took me a while to figure out what `deck-chores-internal` shall mean. what about *any log message … (but not from jobs)*, or similar?
2) for the sake of readability, integers shouldn't be mentioned as possibles values.
3) so far, the notation `stdout` / `stderr` has been used in the docs.